src/UI/item-Card/item-Card.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Meals } from "../../models";
+
+interface ItemCardProps {
+    meal: Meals,
+    addEventHandler: (meal: Meals) => void
+}
+
+export const ItemCard = ({ meal, addEventHandler }: ItemCardProps) => {
+   const image = `./images/${meal.image}`;
+   const addItem = () => {
+       addEventHandler(meal);
```

Comment:
Do you really need to send `meal` as an argument to the event handler? The parent already knows that this item is for the meal specified in the prop. So the prop could be simply called `addItem` with no arguments. This will further simplify your code because you can eliminate this method completely.